### ISSUE TITLE
[Issue]: Should use WAL mode with SQLite

### ISSUE THREAD

**Issue by charlesmunger** [jellyfin/jellyfin#10314]
### Please describe your bug

I started my initial library scan and noticed it was quite slow - profiling with strace:
```
sudo strace -fp <jellyfin pid> -e write,fsync,fdatasync
```
Showed lots of 4kb writes followed by fsync, characteristic of SQLite. 
```
write(287, ""..., 4096) = 4096
...etc
fsync(287)                = 0
fsync(322)                = 0
```
Looking into those FDs, we can see:
```
$ sudo realpath /proc/<pid>/fd/287
/var/lib/jellyfin/data/library.db
$ sudo realpath /proc/<pid>/fd/322
/var/lib/jellyfin/data/library.db-journal
```

Which is using the old rollback journal system. I suspect that switching to WAL mode will address many issues reported by users who claim to have hit the limits of SQLite. You can read more about that [here](https://www.sqlite.org/wal.html).

### Jellyfin Version

10.8.z

### Environment

```markdown
- OS: Ubuntu 22.04
- Linux Kernel: 6.2.0-33-generic
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
We already switched to WAL mode for the next release of Jellyfin.

https://github.com/jellyfin/jellyfin/blob/1b4394199a2f9883cd601bdb8c9d66015397aa52/Emby.Server.Implementations/Data/BaseSqliteRepository.cs#L63

**Comment by charlesmunger** [comment]
That change went in 8 months ago, and maybe it only applied to one of the dbs?

```
$ sqlite3 jellyfin.db 
sqlite> pragma journal_mode;
wal
$ sqlite3 library.db 
sqlite> pragma journal_mode;
delete
$ sqlite3 playback_reporting.db 
sqlite> pragma journal_mode;
delete
```

**Comment by crobibero** [comment]
The change is available in the next release of Jellyfin (10.9), not 10.8.

```
➜ sqlite3 library.db 
SQLite version 3.40.1 2022-12-28 14:03:47
Enter ".help" for usage hints.
sqlite> pragma journal_mode;
wal
```

**Comment by charlesmunger** [comment]
Looking forward to it! Sorry for the noise.
